projects/ontimize-web-ngx/src/lib/components/input/date-range/o-daterange-input.component.ts

```diff
@@ -116,12 +123,14 @@ export class ODateRangeInputComponent extends OFormDataComponent implements OnDe
 
   private momentSrv: MomentService;
   private oTranslate: OTranslateService;
+  private momentDateAdapter: DateAdapter<MomentDateAdapter>;
 
 
   constructor(
     @Optional() @Inject(forwardRef(() => OFormComponent)) form: OFormComponent,
     elRef: ElementRef,
-    injector: Injector
+    injector: Injector,
+    dateAdapter: DateAdapter<OntimizeMomentDateAdapter>
```

Comment:
Instead modifying the constructor to maintain backwards compatibility, retrieve an instance of dateAdapter like it is done with OTranslateService